src/App/PhotosList/PhotosListMini/usePhotoNavigation.js

```
@@ -43,6 +43,12 @@ export function usePhotoNavigation({
 }) {
     const navigateLock = useRef(false);
 
+    // Latest cache snapshot for reads inside async callbacks. Keeping the
+    // read out of setImageCache's deps keeps the callback stable, so
+    // PhotoDisplay effects depending on it don't re-fire per prefetch.
+    const imgCacheMapRef = useRef(imgCacheMap);
+    imgCacheMapRef.current = imgCacheMap;
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

  48 |     // PhotoDisplay effects depending on it don't re-fire per prefetch.
  49 |     const imgCacheMapRef = useRef(imgCacheMap);
> 50 |     imgCacheMapRef.current = imgCacheMap;
     |     ^^^^^^^^^^^^^^^^^^^^^^ Cannot update ref during render
  51 |
  52 |     /**
  53 |      * Lock navigation to prevent rapid-fire navigation

(react-hooks/refs)